src/app/app.routes.ts

```diff
@@ -55,6 +55,13 @@ const protectedRoutes: Routes = [
         (c) => c.TranscriptionRequestsComponent
       ),
   },
+  {
+    path: 'transcription-requests/assign-transcript',
```

Comment:
Doesn't this path need a `:transcriptId`?

Reply:
Will have to think about this one, not sure. I will gather some opinions on this.

Reply:
Please can this question be reserved for the next ticket? https://tools.hmcts.net/jira/browse/DMP-1234